docs/content/concepts/dagster-pipes/aws-glue.mdx

```diff
@@ -46,24 +46,26 @@ Call `open_dagster_pipes` in the Glue job script to create a context that can be
 import boto3
 from dagster_pipes import (
     PipesCliArgsParamsLoader,
+    PipesDefaultMessageWriter,
     PipesS3ContextLoader,
-    PipesS3MessageWriter,
     open_dagster_pipes,
 )
 
 client = boto3.client("s3")
 context_loader = PipesS3ContextLoader(client)
-message_writer = PipesS3MessageWriter(client)
 params_loader = PipesCliArgsParamsLoader()
 
 
 def main():
     with open_dagster_pipes(
         context_loader=context_loader,
-        message_writer=message_writer,
```

Comment:
So this is this our new recommendation? Writing messages to stdout and then doing it re: cloudwatch? What are the cost implications here?

Reply:
Well, AWS GLue jobs emit lots of Glue logs (via stdout/err) by default. 

We will be adding a small amount of dagster logs to these streams. I don't think it matters at all.